test/compress/issue-203.js

```diff
@@ -26,9 +26,7 @@ compress_new_function_with_destruct: {
     }
     expect: {
         Function("a", "[b]", "return a");
-        Function("a", "{bb}", "return a");
-        Function("[[a]]", "[{bb}]", 'return a');
+        Function("a", "{bb:b}", "return a");
+        Function("[[a]]", "[{bb:b}]", 'return a');
```

Comment:
I'm less surprised because my attempt to "improve" that feature got shot down by @mishoo a while back: #1464 :ghost:

https://github.com/mishoo/UglifyJS2/blob/0cc6dedccc1907925cc895ad508fabe4215f4a74/lib/compress.js#L3140-L3196